docker-compose.yml

```diff
@@ -50,9 +49,9 @@ services:
       replicas: 1
       placement:
         constraints:
-          - node.labels.role == opencti-core
+        - node.labels.role == opencti-core
```

Comment:
Not sure which editor you are using but please fix it... otherwise it's going to cause very hard to debug issues. These should be:
```yaml
      placement:
        constraints:
          - node.labels.role == opencti-core
```
and not
```yaml
      placement:
        constraints:
        - node.labels.role == opencti-core
```
no?